modules/core/src/main/scala/Prompt.scala

```diff
@@ -52,9 +62,16 @@ object Prompt:
     override def handler(
         terminal: Terminal,
         output: Output,
-        colors: Boolean
+        colors: Boolean,
+        windowSize: Int = 10
```

Comment:
Is there a reason to not push windowSize parameter into just the prompts that need it? namely single choice and multi-choice.

Unlike colors, it's not a universal property shared by all prompts, it's only really applicable to single/multi choice and (future) search prompt